zmon_aws_agent/aws.py

```diff
@@ -739,7 +739,7 @@ def get_apps_from_entities(instances, account, region):
     return applications
 
 
-def get_limits(region, acc, apps, elbs):
+def get_limits(region, acc, apps, elbs, entities):
```

Comment:
I suggest we get the existing entity first (with ``type == 'aws_limits'``) and then use the value in the ``limits`` dict below. That will save us the condition at the end. Another part to cover would be the other limits, like ``rds`` and ``certificates``, ``instance profiles`` etc..., having the existing entity would help us to keep those values in case of error and just use ``entity.update(limits)`` to overwrite.